app/i18n/participation/fr.js

```diff
@@ -0,0 +1,217 @@
+    "stepsAndProgress": "Étapes et progrès",
+    "educationalPublic": "Éducatif/public",
+    "governmentState": "Gouvernement/État",
+    "private": "Privé",
+    "other": "Autre",
+    "print": "Imprimer",
+    "newsAgencyService": "Agence/service de presse",
+    "radio": "Radio",
+    "photoVisual": "Photo/visuel",
+    "television": "Télévision",
+    "independentBroadcastOrFilmProduction": "Production indépendante de diffusion ou de film",
+    "weeklyPublication": "Parution hebdomadaire",
+    "webPublication": "Publication Web",
+    "onlineJournal": "Revue en ligne",
+    "blog": "Blog",
+    "otherOnlineMedia": "Autres médias en ligne",
+    "freelanceJournalist": "Journaliste indépendant",
+    "zip": "Fermeture éclair",
+    "powerPointPpt": "PowerPoint (ppt)",
+    "powerPointPptx": "PowerPoint (pptx)",
+    "excelXls": "Excel (xls)",
+    "excelXlsx": "Excel (xlsx)",
+    "wordDoc": "Word (doc)",
+    "wordDocx": "Mot (docx)",
+    "pdf": "PDF",
+    "mar": "Demande d'accréditation média",
+    "requirements": "Exigences",
+    "organization": "Organisation",
+    "contacts": "Contacts",
+    "participantS": "Participant(s)",
+    "finished": "Fini",
+    "addParticipant": "Ajouter un participant",
+    "name": "Nom",
+    "positionEmail": "Poste/E-mail",
+    "edit": "Modifier",
+    "previous": "Précédent",
+    "submit": "Soumettre",
+    "identification": "Identification",
+    "nameOfTheOrganization": "Nom de l'organisation",
+    "nameIsRequired": "Le nom est requis.",
+    "acronym": "Acronyme",
+    "sector": "Secteur",
+    "typeIsRequired": "le type est requis.",
+    "typeOfMediaS": "Type de média",
+    "mediumIsRequired": "un médium est nécessaire.",
+    "address": "Adresse",
+    "contact": "Contact",
+    "phone": "Téléphone",
+    "phoneNumberIsRequired": "Le numéro de téléphone est requis.",
+    "pleaseMatchPattern_1_514_288_2220": "Veuillez faire correspondre le modèle [+1 514.288.2220]",
+    "email": "E-mail",
+    "emailIsRequired": "Un e-mail est requis.",
+    "emailIsNotValid": "L'email n'est pas valide.",
+    "emailCCs": "Envoyer des CC par courrier électronique",
+    "website": "Site web",
+    "websiteUrlNotCorrectFormatSuchAsHttpsWwwCbdInt": "l'URL du site Web n'est pas un format correct tel que https://www.cbd.int",
+    "language": "Langue",
+    "languageIsRequired": "La langue est requise.",
+    "fileUploads": "Téléchargements de fichiers",
+    "letterOfAssignment": "Lettre de mission",
+    "assignmentLetterIsRequired": "Une lettre de mission est requise.",
+    "attachments": "Pièces jointes",
+    "next": "Suivant",
+    "thankYouForYourMediaAccreditationSubmission": "Merci pour votre demande d'accréditation média. Veuillez noter que votre demande sera traitée dans un délai de 7 jours dans des circonstances normales. Si vous ne recevez pas de réponse dans ce délai, veuillez contacter l'équipe média au .",
+    "secretariatCbdInt": "secrétariat@cbd.int",
```

Comment:
email value should not be translated 
```suggestion
    "secretariatCbdInt": "secretariat@cbd.int",
```